### ISSUE TITLE
HTML entities no longer parsed

### ISSUE THREAD

**Issue by jeffblake** [k0kubun/hamlit#27]
This
`&times;`

used to correctly render the html entity, now it doesn't. `"&times;"` is outputted to the screen.

Applies to all entities.

**Comment by k0kubun** [comment]
Fixed and released as v1.6.0.

**Comment by jeffblake** [comment]
Fast fix! Thanks.

**Comment by eagletmt** [comment]
Actually this behavior is very so inconsistent and ad-hoc.
https://gist.github.com/eagletmt/8bf43f26df237ac331d5

**Comment by k0kubun** [comment]
> Actually this behavior is very so inconsistent and ad-hoc.

I think so too. :p
I keep hamlit to be compatible with latest haml, but it may be hard to maintain...

**Comment by leesmith** [comment]
Using hamlit 1.5.4, this snippet outputs the html "&times" entity correctly:

```
%button.close{'aria-hidden' => 'true', 'data-dismiss' => 'alert', type: 'button'} &times;
```

With hamlit 1.5.5+, I have to do this, otherwise it prints "&times;":

```
%button.close{'aria-hidden' => 'true', 'data-dismiss' => 'alert', type: 'button'}
  \&times;
```

Is this expected behavior?  Thanks for any help!

**Comment by k0kubun** [comment]
@leesmith It's my mistake. Thank you for reporting regression with versions. Fixed in v1.6.3.

```sh
$ cat a.haml
%button.close{'aria-hidden' => 'true', 'data-dismiss' => 'alert', type: 'button'}
  &times;
%button.close{'aria-hidden' => 'true', 'data-dismiss' => 'alert', type: 'button'} &times;

$ hamlit render a.haml 
<button aria-hidden='true' class='close' data-dismiss='alert' type='button'>
&times;
</button>
<button aria-hidden='true' class='close' data-dismiss='alert' type='button'>&times;</button>
```

**Comment by leesmith** [comment]
Thanks @k0kubun!
